cmake/public/xpu.cmake

```diff
@@ -0,0 +1,70 @@
+# XPU backend stacks (compiler, runtime, libraries, tools)
+#
+# PYTORCH_FOUND_XPU
+# PYTORCH_SYCL_LIBRARIES
+# SYCL_INCLUDE_PATHS
```

Comment:
Can you elaborate more on these cmake variables, why you list them here? It also makes to add comments for a high-level summary what this cmake file does and what it inputs and what it outputs, etc.